### PR TITLE
Add page about how to create code releases

### PR DESCRIPTION
**Pull request by christianhueserhzdr**
This page provides useful aspects to be considered when creating code releases.

### CODE REVIEW

**Comment by dgarijo** [comment]
@christianhueserhzdr I understand thi PR is about the task in here: https://docs.google.com/spreadsheets/d/1TMzE5umQ2rkS9POAmhlAPNptBKin4eC3YW3YbijGci4/edit?gid=0#gid=0. However, you are not following the template we have set up for this task, which is found here: https://github.com/EVERSE-ResearchSoftware/RSQKit/blob/main/pages/your_tasks/TEMPLATE_your_tasks.md. Would you mind adapting it to the template? All tasks should be consistent.

While much of the content is there and it is important, I think some of the contents should link to other pages. For example, creating a citation.cff is a task page on its own. Thanks again for your contribution!

File: pages/research_software/software_releases.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+If no license is given all rights are reserved by the copyright holders and no rights are granted to others.
+It is recommended to release your software as open source software.
+In brief, for open source software rights like usage, modifications and distribution are granted to the users of your software.
+This involves choosing an appropriate open source software license from the list of [OSI approved licenses](https://opensource.org/licenses) beforehand.
+Each file in the release needs to have a license statement which can be verified by tools like [REUSE](https://reuse.software/).
+On top, the repository should include a file like `License.md` that contains the license text of the license used.
+
+## Citations
+
+You might also want to provide a file in the [Citation File Format (CFF)](https://citation-file-format.github.io/) like `CITATION.cff` to give citation information about your software and make sure that other can cite your software releases properly.
```
Comment (dgarijo) [inline pages/research_software/software_releases.md:38]: We do not have a task for this, but we will. I suggest not repeating this too.

File: pages/research_software/software_releases.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Changelog
+
+Semantic Versioning gives you a rough idea about the impact the release might have and the severity of changes when moving from one version to another.
+Nonetheless, it is inevitable to provide a standarised [Changelog](https://keepachangelog.com/en/1.1.0/) to users that explains in detail which changes where applied to a particular release.
+You can generated Changelogs from your commit messages with Changelog Generators like [git-cliff](https://www.conventionalcommits.org/en/v1.0.0/).
+Commit messages need to follow specific conventions.
+Writing good commit messages is essential for users to know what has been changed and why.
+It is recommended to follow common advice about writing [good commit messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/).
```
Comment (dgarijo) [inline pages/research_software/software_releases.md:47]: We should probably mention other versioning schemes, such as calendar-based.
Committing is not about doing software releases, right?

File: pages/research_software/software_releases.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+# Software Releases
+
+## Introdution
+
+Developing software is an iterative process which also involves the creation software releases at particular points in time.
+In a software development life cycle this phase is located between testing & quality assurance and maintenance (see diagram below).
+
+![Release phase in software development life cycle](../../assets/img/release-in-sw-dev-life-cycle.png)
```
Comment (dgarijo) [inline pages/research_software/software_releases.md:12]: I think the image is nice, but shouldn't it be consistent with the content in the Reference model? We currently have another image.

**Comment by dgarijo** [comment]
Looking good! Anything missing?

**Comment by christianhueserhzdr** [comment]
> Looking good! Anything missing?

For the time being I have nothing to add to this page. I'll remove the draft state.

**Review by dgarijo — APPROVED** [review]

**Pull request merged**